public/doc/openapi-3-v1.0.0.json

```diff
@@ -962,27 +962,7 @@
           "content": {
             "application/json": {
               "schema": {
-                "type": "object",
-                "properties": {
-                  "query": {
-                    "type": "string",
-                    "description": "The GraphQL query",
-                    "default": "{}"
-                  },
-                  "operationName": {
-                    "type": "string",
-                    "description": "If the Query contains several named operations, the operationName controls which one should be executed",
-                    "default": ""
-                  },
-                  "variables": {
-                    "type": "object",
-                    "description": "Optional Query variables",
-                    "nullable": true
-                  }
-                },
-                "required": [
-                  "query"
-                ]
+                "$ref": "#/components/schemas/GraphQLRequest"
```

Comment:
@abellotti actually generator seems to be deleting this line now

Reply:
@abellotti I think you need to update the `manageiq-api-common` dependency to 1.0.0 to pull in those changes, right?

Reply:
I missed this here somehow https://github.com/ManageIQ/manageiq-api-common/pull/120